src/core/execute.h

```diff
@@ -302,6 +302,7 @@ struct ExecContext {
         ProcSubset proc_subset;    /* subset= */
 
         int private_mounts;
+        int memory_ksm;
```

Comment:
initialize this field to -1 in exec_context_init

Reply:
Done.